addons/web_editor/static/src/js/wysiwyg/widgets/link.js

```diff
@@ -243,6 +237,22 @@ const Link = Widget.extend({
         }
         return link;
     },
+    /**
+     * Public method to focus the url input.
+     * Called recursively until the input is focused.
+     */
+    async focusUrl() {
+        if (this.$el) {
+            const [$firstInput] = this.$el.find('input:visible:first');
```

Comment:
- `$firstInput `is named with a `$` while it does not contain a jQuery instance
- Don't use jQuery anyway